common/setups/rasr/util/flow/tensorflow.py

```diff
@@ -0,0 +1,86 @@
+__all__ = ["get_tf_flow", "add_tf_flow_to_base_flow"]
+
+from typing import List, Optional, Union
+
+from sisyphus import tk
+
+import i6_core.rasr as rasr
+import i6_core.returnn as returnn
+
+
+def get_tf_flow(
```

Comment:
Do we want to have this in `i6_core` instead? This seems to be a very fundamental function.

Reply:
You mean the tensorflow flow node configuration? IMO, this should live under `i6_experiments`

Reply:
Why? All other flows are in `i6_core`, so I thought this should also be there.